cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.js

```diff
@@ -302,6 +305,11 @@ function LogViewerPreviewController ($scope, $window, LogViewerStore, myPreviewL
     });
   };
 
+  vm.toggleSystemLogs = () => {
+    vm.includeSystemLogs = !vm.includeSystemLogs;
+    startTimeRequest();
```

Comment:
if it doesn't have fromOffset, then the entire implementation of the log viewer is wrong right? how do we fetch the next set of logs?